### PR TITLE
Add demo scripts

### PR DESCRIPTION
**Pull request by falkecarlsen**
Note that these scripts are tailored for a specific machines configuration

### CODE REVIEW

File: demo/demo-netgenerate.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+rm "out/cities/*" || true
+
+echo "====================="
+echo ""
+echo " NETGENERATE "
+echo ""
+echo "====================="
+
+./in/generateNetLinux.sh
```
Comment (jenrik) [inline demo/demo-netgenerate.sh:15]: `generateNetLinux.sh` doesn't use the utility from the a docker contains, and instead assumes you have the tool in `SUMO_HOME/bin/netgenerate`

**Review by jenrik — CHANGES_REQUESTED** [review]
`demo/demo-netgenerate.sh` doesn't work, and fails with an error from ActivityGen

File: demo/demo-netgenerate.sh (hunk 1 above)
Comment (falkecarlsen) [reply demo/demo-netgenerate.sh:15]: That's hard for me to test, since it depends on your setup.

**Comment by jenrik** [comment]
not suited for public release

**Pull request closed without merging**